.circleci/config.yml

```diff
@@ -8,47 +8,65 @@ runOnDocker: &runOnDocker
   docker:
     - image: circleci/golang:1.11.4
 
+restoreGoModCache: &restoreGoModCache
```

Comment:
it would be nice to not mix camelCase with underscores

everywhere else in config.yml `_` notation is used so it will be goo to change names

Reply:
That's not totally true. All yml ref are camelCase (Eg: `&runOnMachine`).
But I agree, it should be only one style. I will update.

Reply:
Done